Lesson 32 HomeWork/Lesson 32 HomeWork/ContentView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ContentView.swift
+//  Lesson 32 HomeWork
+//
+//  Created by Ernest Mihasenko on 28.06.22.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        Text("Hello, world!")
+            .padding()
+            .backGroundModifier()
+            .fontViewModifier()
+            .upperLabelModifier()
+    }
+}
+
+let purpleColor = Color.purple
+let brownColor = Color.brown
+let cornerRadius: CGFloat = 50
+let customFont: Font = .custom("Marker Felt", size: 50)
+let modifierString = "upper label"
```

Comment:
You made just these properties global but you still don't pass them as parameters to your view modifiers. There's no way to use the same modifier twice with your current implementation. You need to create properties inside your modifiers and a constructor will be created automatically because your modifiers are structs. Please, take another look at our lesson project on this topic